### PR TITLE
Clean up code a bit

### PR DESCRIPTION
**Pull request by BBlackwo**

### CODE REVIEW

File: core/util/runTest.js [hunk 1]
```diff
@@ -44,30 +44,28 @@ module.exports.runTests = async testSuite => {
 };
 
 const getMethod = scenario => {
-  return scenario.request.method
-    ? scenario.request.method
-    : configs.defaultMethod
-    ? configs.defaultMethod
-    : DEFAULT_METHOD;
+  return scenario.request.method || configs.defaultMethod || DEFAULT_METHOD;
```
Comment (BBlackwo) [inline core/util/runTest.js:47]: I think this is what you wanted?

File: core/util/runTest.js [hunk 2]
```diff
@@ -3,7 +3,7 @@ const Promise = require("bluebird");
 const fs = require("fs");
 const path = require("path");
 const chalk = require("chalk");
-const log = console.log;
+const { log } = console;
 
 const DEFAULT_METHOD = "get";
 let configs;
```
Comment (BBlackwo) [inline core/util/runTest.js:9]: I'm not a big fan of this, I reckon passing the config to each method is better rather than assigning to a global one

**Comment by BBlackwo** [comment]
Btw `Promise.map()` is really cool :) I've never seen that before :+1:

File: core/util/runTest.js (hunk 2 above)
Comment (marcdacz) [reply core/util/runTest.js:9]: Will update this as param instead of global var

File: core/util/runTest.js (hunk 1 above)
Comment (marcdacz) [reply core/util/runTest.js:47]: Nice!

**Pull request merged**